### DISCUSSION TITLE
Group output metadata per segment

### DISCUSSION THREAD

**Discussion by skezu** [speechmatics/community#13] [General chat]
So with the default settings the output is split words per words. However I want, in the JSON to have the same segmentation as the .srt output. I've looked through the doc and I can't seem to find it :/
Is it possible ? If yes how to do it 🙏
What I have :

What I want :
```json
"results": [
    {
      "alternatives": [
        {
          "confidence": 0.95,
          "content": "Just this",
          "language": "en",
          "speaker": "UU"
        }
      ],
      "end_time": 1.44,
      "start_time": 0.9,
      "type": "sentence"
    }
  ]
```

```json
"results": [
    {
      "alternatives": [
        {
          "confidence": 0.9,
          "content": "Just",
          "language": "en",
          "speaker": "UU"
        }
      ],
      "end_time": 1.07,
      "start_time": 0.9,
      "type": "word"
    },
    {
      "alternatives": [
        {
          "confidence": 1,
          "content": "this",
          "language": "en",
          "speaker": "UU"
        }
      ],
      "end_time": 1.44,
      "start_time": 1.11,
      "type": "word"
    },
    {
      "alternatives": [
        {
          "confidence": 1,
          "content": ".",
          "language": "en",
          "speaker": "UU"
        }
      ],
      "attaches_to": "previous",
      "end_time": 273.64,
      "start_time": 273.64,
      "type": "punctuation"
    }
  ]
```

**Comment by nickgerig** [comment]
Hi @skezu we don't currently expose this info in JSON unfortunately. Can you give us some insight as to why you would need this info in JSON rather than SRT?

**Reply by skezu** [reply]
I need this information in JSON format because I want a **confidence score per segment**, not per word (it's a data that's not available in the srt file). It would greatly benefit us if we could receive the confidence score per segment directly in the API response, as it would allow us to **review** the transcriptions.

**Reply by nickgerig** [reply]
We only have word level confidence so you'd have to infer sentence level confidence from that.

**Reply by skezu** [reply]
Yeah that's what I did, it's unfortunate that we can't get that directly from the API's response tho 🥲

**Reply by nickgerig** [reply]
Thanks for providing some context, we are aware of the use case now.

**Comment by ProzegachIvanWork** [comment]
is there a request that will indicate the list of available languages?
